.env.example

```
@@ -29,3 +29,7 @@ BBB_HEALTH_PROBE_INTERVAL_SECONDS=30  # Health probe interval when all workers i
 DEBUG=true                  # Enable debug logging
 LOG_LEVEL=debug            # debug, info, warn, or error
 RUN_INTEGRATION_TESTS=true # Enable integration tests
+
+# Webflow
+WEBFLOW_CLIENT_ID=your_client_id_here
+WEBFLOW_CLIENT_SECRET=your_client_secret_here
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)